### PR TITLE
Dev indirect direct message counts

### PR DESCRIPTION
**Pull request by uil9**

### CODE REVIEW

File: processing-status-api-function-app/test/load-testing/__pycache__/reports.cpython-312.pyc
Comment (mkrystof) [inline processing-status-api-function-app/test/load-testing/__pycache__/reports.cpython-312.pyc:1]: Can you remove this file from the PR?  Doesn't seem like it belongs.

**Review by mkrystof — CHANGES_REQUESTED** [review]
Just the one request, otherwise looks good.

File: docs/processing-status-openapi-spec.yaml [hunk 1]
```diff
@@ -759,6 +759,162 @@ paths:
...
+          example: 20231020T14103000Z
+        - name: date_end
+          in: query
+          schema:
+            type: string
+          description: >-
+            End date of filter.  Format is YYYYMMDDTHHMMSSZ (UTC format,
+            exclusive).  Defaults to today.
+          example: 20231030T14103000Z
+      responses:
```
Comment (mkrystof) [inline docs/processing-status-openapi-spec.yaml:900]: Would you please add days_interval here as well?

Comment (uil9) [reply docs/processing-status-openapi-spec.yaml:924]: Added days_interval

**Pull request merged**